### PR TITLE
HOT-2147 Add SSB data to submit screening

### PR DESCRIPTION
**Pull request by OlegKorniichuk**
SSB

## Description
Add SSB to submit screening

## Jira Issue link
https://osi-cwds.atlassian.net/browse/HOT-2147

## Tests
- [x] I have included unit tests 
- [ ] I have included functional tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the google code style of this project.
- [ ] I have ran all tests for the project.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build, to check SonarQube after the build completes, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: src/main/java/gov/ca/cwds/rest/api/domain/SafelySurrenderedBabiesIntakeApi.java [hunk 1]
```diff
@@ -26,22 +28,28 @@
...
-  @ApiModelProperty(required = false, readOnly = false, value = "", example = "1234",
+  @ApiModelProperty(required = true, readOnly = false, value = "", example = "1592",
       notes = "The code for relationship")
+  @NotEmpty
   private String relationToChild;
 
   @JsonProperty("bracelet_id")
-  @ApiModelProperty(example = "1234")
+  @ApiModelProperty(required = true, readOnly = false, example = "1234")
+  @Size(max = 10)
```
Comment (saleemis-cwds) [inline src/main/java/gov/ca/cwds/rest/api/domain/SafelySurrenderedBabiesIntakeApi.java]: Do we have to do this validation?

File: src/main/java/gov/ca/cwds/rest/api/domain/SafelySurrenderedBabiesIntakeApi.java [hunk 2]
```diff
@@ -26,22 +28,28 @@
...
   private String braceletId;
 
   @JsonProperty("parent_guardian_given_bracelet_id")
-  @ApiModelProperty(example = "U")
+  @ApiModelProperty(required = true, readOnly = false, example = "U")
+  @NotEmpty
   @Size(max = 1)
+  @OneOf(value = {"A", "N", "U", "Y"})
```
Comment (saleemis-cwds) [inline src/main/java/gov/ca/cwds/rest/api/domain/SafelySurrenderedBabiesIntakeApi.java]: Do we have to do this validation? I think we said, we should allow invalid data while screening.

File: src/main/java/gov/ca/cwds/rest/api/domain/SafelySurrenderedBabiesIntakeApi.java (hunk 1 above)
Comment (OlegKorniichuk) [reply src/main/java/gov/ca/cwds/rest/api/domain/SafelySurrenderedBabiesIntakeApi.java]: Validation was moved to drools

File: src/main/java/gov/ca/cwds/rest/api/domain/SafelySurrenderedBabiesIntakeApi.java (hunk 2 above)
Comment (OlegKorniichuk) [reply src/main/java/gov/ca/cwds/rest/api/domain/SafelySurrenderedBabiesIntakeApi.java]: Validation was moved to drools

**Pull request merged**